### PR TITLE
TI Site requirements

### PR DESCRIPTION
**Pull request by SergejMuhic**
1. initial commit with Name and Bridge requirements

### CODE REVIEW

**Review by larswik — APPROVED** [review]
I think that this is a good start that we can work with.

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 1]
```diff
@@ -101,6 +101,27 @@
...
 		</DocConceptRoot>
+		<DocConceptRoot Name="Site" UniqueId="5b07a4ce-3e5c-46d8-b52b-816501841e44">
+			<Documentation>Data sets shall include at least one site. The site may define existing and proposed terrain within vicinity of the bridge project and contain alignment(s), bridge(s) and potentially other structures such as the road/rail supported by the bridge. Boring points may be included as objects contained within the site.</Documentation>
+			<ApplicableEntity xsi:type="DocEntity" xsi:nil="true" href="IfcSite" />
+			<Concepts>
+				<DocTemplateUsage Name="Name" UniqueId="3fc89f97-ae7f-48fd-bf86-c763a731a9f0">
+					<Documentation>Name of the site for referencing purposes.</Documentation>
+					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Object_User_Identity_31di66dFr7AJKdxDG2D1A" />
+					<Items>
+						<DocTemplateItem UniqueId="e76f143a-90eb-4b24-bde9-ae14d9e78e18" RuleParameters="ObjectName=HDMI Plattrambro;" />
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml:110]:
```suggestion
						<DocTemplateItem UniqueId="e76f143a-90eb-4b24-bde9-ae14d9e78e18" RuleParameters="ObjectName[Value]='HDMI Plattrambro'" />
```

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 2]
```diff
@@ -101,6 +101,27 @@
...
+					</Items>
+				</DocTemplateUsage>
+				<DocTemplateUsage Name="Bridge" UniqueId="7554fcb5-f806-44b6-aabf-b741ca557728">
+					<Documentation>Bridge structure(s) within site.</Documentation>
+					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Spatial_Decomposition_1cVuH3xCvAZQOan_h15FW" />
+					<Items>
+						<DocTemplateItem UniqueId="57319180-2ea6-4688-b8e4-f04b377802b0" RuleParameters="RelatedObjects=IfcBridge;" />
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]:
```suggestion
						<DocTemplateItem UniqueId="57319180-2ea6-4688-b8e4-f04b377802b0" RuleParameters="RelatedObjects[Type]='IfcBridge'" />
```

**Review by pjanck — APPROVED** [review]
Updating to 1.1 syntax for parameter strings. I am unsure how this is handled internally, though.

**Comment by pjanck** [comment]
> Updating to 1.1 syntax for parameter strings.

Disregard this.

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 3]
```diff
@@ -91,8 +91,38 @@
...
+					</Items>
+				</DocTemplateUsage>
+				<DocTemplateUsage Name="Surface Existing" UniqueId="ac589648-5a70-4f3e-99d2-4ce5d30dba1d">
+					<Documentation>The ground surface of the site indicating existing conditions.</Documentation>
+					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Spatial_Container_1XtGZjG59LPCtYlqeIWvl" />
+					<Items>
+						<DocTemplateItem UniqueId="875a4dfb-183b-4bda-8eb8-2c861cbb0506" RuleParameters="Type=IfcGeomodel;" />
+						<DocTemplateItem UniqueId="2fa38e7e-3f85-4c30-932c-83431815087f" RuleParameters="Type=IfcGeotechnicalStratum;" />
+					</Items>
+				</DocTemplateUsage>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]: Top `DocTemplateUsage` has an attribute `Operator="or"`, the bottom doesn't. Intentional?

Comment (SergejMuhic) [reply IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]: Nope, corrected.

**Review by pjanck — CHANGES_REQUESTED** [review]
I am missing `Spatial containment` on:

- boring point
- geomodel
- geotechnical stratum

Are these not required?

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 4]
```diff
@@ -1,6 +1,72 @@
...
+							<Concepts>
+								<DocTemplateUsage Name="Properties" UniqueId="9c755010-4bbe-44f7-a6e1-0130efec88fe">
+									<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Single_Value_1cLVRGAQX7k8yzoVpfoQOW" />
+								</DocTemplateUsage>
+							</Concepts>
+						</DocTemplateItem>
+					</Items>
+				</DocTemplateUsage>
+				<DocTemplateUsage Name="Product Local Placement" UniqueId="768316c5-4071-4279-817a-d1679cf4c55b">
+					<Documentation>Boreholes shall be placed localy with _IfcLocalPlacement_.</Documentation>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]:
```suggestion
					<Documentation>Boreholes shall be placed locally with _IfcLocalPlacement_.</Documentation>
```

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 5]
```diff
@@ -1,6 +1,72 @@
...
+			<Concepts>
+				<DocTemplateUsage Name="Product Local Placement" UniqueId="9638f81d-7179-45f7-b04b-699960684a0a">
+					<Documentation>Surface Proposed shall be locally placed in the Site.</Documentation>
+					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Product_Local_Placement_3Bw5jVUH9AGukt7cEGB9j" />
+					<Items>
+						<DocTemplateItem UniqueId="e23826a6-9eb2-46cc-ad7e-24c3f34ad57e" RuleParameters="RelativeToElement=IfcSite;ElementName=HDMI Plattrambro;" />
+					</Items>
+				</DocTemplateUsage>
+				<DocTemplateUsage Name="Geometry" UniqueId="a87476ed-13ba-43e9-964b-6c4d0b88869e">
+					<Documentation>Surface Proposed shall be represented as a 3D surface tesselation _IfcTriangulatedIrregularNetwork_ geometric items.</Documentation>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]:
```suggestion
					<Documentation>Surface Proposed shall be represented as a 3D surface tessellation _IfcTriangulatedIrregularNetwork_ geometric item.</Documentation>
```

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 6]
```diff
@@ -1,6 +1,72 @@
...
+				<DocTemplateUsage Name="Geometry" UniqueId="8057968e-6351-4ee9-88aa-a7791000fe4f">
+					<Documentation>Surface Proposed shall be represented as a 3D surface tesselation _IfcTriangulatedIrregularNetwork_ geometric items.</Documentation>
+					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Surface_Tessellation_Geometry_0gcN00aHT7P84eQ4tB7HA6" />
+					<Items>
+						<DocTemplateItem UniqueId="c4746c74-b98a-436e-80d7-48789063752e" RuleParameters="Items=IfcTriangulatedIrregularNetwork;" />
+					</Items>
+				</DocTemplateUsage>
+			</Concepts>
+			<ApplicableTemplate xsi:type="DocTemplateDefinition" UniqueId="f3811f64-c132-4240-89da-fa649f266b5d" Type="IfcGeomodel" />
+		</DocConceptRoot>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]: Should we already account for https://github.com/bSI-InfraRoom/IFC-Specification/pull/651 and use _IfcGeoScienceModel_?

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 7]
```diff
@@ -34,6 +34,32 @@
...
+							<Concepts>
+								<DocTemplateUsage Name="Properties" UniqueId="9c755010-4bbe-44f7-a6e1-0130efec88fe">
+									<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Single_Value_1cLVRGAQX7k8yzoVpfoQOW" />
+								</DocTemplateUsage>
+							</Concepts>
+						</DocTemplateItem>
+					</Items>
+				</DocTemplateUsage>
+				<DocTemplateUsage Name="Product Local Placement" UniqueId="768316c5-4071-4279-817a-d1679cf4c55b">
+					<Documentation>Boreholes shall be placed localy with _IfcLocalPlacement_.</Documentation>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]:
```suggestion
					<Documentation>Boreholes shall be placed locally with _IfcLocalPlacement_.</Documentation>
```

Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]:
```suggestion
					<Documentation>Boreholes shall be placed locally with _IfcLocalPlacement_.</Documentation>
```

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 8]
```diff
@@ -73,5 +99,137 @@
...
+						<DocTemplateItem UniqueId="57319180-2ea6-4688-b8e4-f04b377802b0" RuleParameters="RelatedObjects=IfcBridge;SpatialElementName=2-2241-1;" />
+					</Items>
+				</DocTemplateUsage>
+				<DocTemplateUsage Name="Project" UniqueId="a9a7d68d-c84f-4fae-b753-f975c296bef8">
+					<Documentation>The Site shall be contained in a project.</Documentation>
+					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Spatial_Composition_2C3zBtSRjEReR30ApKl5A" />
+					<Items>
+						<DocTemplateItem UniqueId="6effeb6a-ec4c-4587-8d33-df811fca5cfd" RuleParameters="RelatingObject=IfcProject;ProjectName=8448590;" />
+					</Items>
+				</DocTemplateUsage>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]: Capitalization of site/Site is inconsistent in documentation nodes.

```suggestion
				<DocTemplateUsage Name="Bridge" UniqueId="7554fcb5-f806-44b6-aabf-b741ca557728">
					<Documentation>Bridge structure(s) within site.</Documentation>
					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Spatial_Decomposition_1cVuH3xCvAZQOan_h15FW" />
					<Items>
						<DocTemplateItem UniqueId="57319180-2ea6-4688-b8e4-f04b377802b0" RuleParameters="RelatedObjects=IfcBridge;SpatialElementName=2-2241-1;" />
					</Items>
				</DocTemplateUsage>
				<DocTemplateUsage Name="Project" UniqueId="a9a7d68d-c84f-4fae-b753-f975c296bef8">
					<Documentation>The site shall be contained in a project.</Documentation>
					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Spatial_Composition_2C3zBtSRjEReR30ApKl5A" />
					<Items>
						<DocTemplateItem UniqueId="6effeb6a-ec4c-4587-8d33-df811fca5cfd" RuleParameters="RelatingObject=IfcProject;ProjectName=8448590;" />
					</Items>
				</DocTemplateUsage>
```

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 9]
```diff
@@ -73,5 +99,137 @@
...
+			</Concepts>
+			<ApplicableTemplate xsi:type="DocTemplateDefinition" UniqueId="9a30959e-16a1-442c-84ca-7521ed88a2e7" Type="IfcSite" />
+		</DocConceptRoot>
+		<DocConceptRoot Name="Surface Existing Geomodel" UniqueId="7db57e42-f8fb-454c-aa0d-5f803c837b8c">
+			<ApplicableEntity xsi:type="DocEntity" xsi:nil="true" href="IfcGeomodel" />
+			<Concepts>
+				<DocTemplateUsage Name="Product Local Placement" UniqueId="437748ef-61eb-485d-92d2-7cdc615a1805">
+					<Documentation>Surface Proposed shall be locally placed in the Site.</Documentation>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]:
```suggestion
				<DocTemplateUsage Name="Product Local Placement" UniqueId="437748ef-61eb-485d-92d2-7cdc615a1805">
					<Documentation>Surface Proposed shall be locally placed in the site.</Documentation>
```

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 10]
```diff
@@ -73,5 +99,137 @@
...
+					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Product_Local_Placement_3Bw5jVUH9AGukt7cEGB9j" />
+					<Items>
+						<DocTemplateItem UniqueId="db040752-8fee-45c7-ba17-78eeaa936a13" RuleParameters="RelativeToElement=IfcSite;ElementName=HDMI Plattrambro;" />
+					</Items>
+				</DocTemplateUsage>
+				<DocTemplateUsage Name="Geometry" UniqueId="35622964-9cc2-47ae-aeb1-e465fcb77330">
+					<Documentation>Surface Proposed shall be represented as a 3D surface tesselation _IfcTriangulatedIrregularNetwork_ geometric items.</Documentation>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]:
```suggestion
				<DocTemplateUsage Name="Geometry" UniqueId="35622964-9cc2-47ae-aeb1-e465fcb77330">
					<Documentation>Surface Proposed shall be represented as a 3D surface tessellation _IfcTriangulatedIrregularNetwork_ geometric items.</Documentation>
```

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 11]
```diff
@@ -73,5 +99,137 @@
...
+					</Items>
+				</DocTemplateUsage>
+			</Concepts>
+			<ApplicableTemplate xsi:type="DocTemplateDefinition" UniqueId="f3811f64-c132-4240-89da-fa649f266b5d" Type="IfcGeomodel" />
+		</DocConceptRoot>
+		<DocConceptRoot Name="Surface Proposed Geotechnical Stratum" UniqueId="49dde5fa-446a-4d74-8b37-dbee334423c7">
+			<Documentation>Ground surface of the Site indicating final conditions modelled as _IfcGeotechnicalStratum_.</Documentation>
+			<Concepts>
+				<DocTemplateUsage Name="Product Local Placement" UniqueId="9638f81d-7179-45f7-b04b-699960684a0a">
+					<Documentation>Surface Proposed shall be locally placed in the Site.</Documentation>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml:217]:
```suggestion
					<Documentation>Surface Proposed shall be locally placed in the site.</Documentation>
```

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 12]
```diff
@@ -73,5 +99,137 @@
...
+					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Product_Local_Placement_3Bw5jVUH9AGukt7cEGB9j" />
+					<Items>
+						<DocTemplateItem UniqueId="e23826a6-9eb2-46cc-ad7e-24c3f34ad57e" RuleParameters="RelativeToElement=IfcSite;ElementName=HDMI Plattrambro;" />
+					</Items>
+				</DocTemplateUsage>
+				<DocTemplateUsage Name="Geometry" UniqueId="a87476ed-13ba-43e9-964b-6c4d0b88869e">
+					<Documentation>Surface Proposed shall be represented as a 3D surface tesselation _IfcTriangulatedIrregularNetwork_ geometric items.</Documentation>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]:
```suggestion
					<Documentation>Surface Proposed shall be represented as a 3D surface tessellation _IfcTriangulatedIrregularNetwork_ geometric items.</Documentation>
```

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 13]
```diff
@@ -73,5 +99,137 @@
...
+			<Concepts>
+				<DocTemplateUsage Name="Product Local Placement" UniqueId="52718638-0df6-4826-a4ae-cc5ec683a410">
+					<Documentation>Surface Proposed shall be locally placed in the Site.</Documentation>
+					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Product_Local_Placement_3Bw5jVUH9AGukt7cEGB9j" />
+					<Items>
+						<DocTemplateItem UniqueId="21fccf7f-21bc-47b2-8144-ac2d7a3134fd" RuleParameters="RelativeToElement=IfcSite;ElementName=HDMI Plattrambro;" />
+					</Items>
+				</DocTemplateUsage>
+				<DocTemplateUsage Name="Geometry" UniqueId="8057968e-6351-4ee9-88aa-a7791000fe4f">
+					<Documentation>Surface Proposed shall be represented as a 3D surface tesselation _IfcTriangulatedIrregularNetwork_ geometric items.</Documentation>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]:
```suggestion
					<Documentation>Surface Proposed shall be represented as a 3D surface tessellation _IfcTriangulatedIrregularNetwork_ geometric items.</Documentation>
```

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 14]
```diff
@@ -73,5 +99,137 @@
...
+					</Items>
+				</DocTemplateUsage>
+			</Concepts>
+			<ApplicableTemplate xsi:type="DocTemplateDefinition" UniqueId="6ce8a1b2-8cb9-41f4-a94c-60809e3bbe47" Type="IfcGeotechnicalStratum" />
+		</DocConceptRoot>
+		<DocConceptRoot Name="Surface Proposed Geomodel" UniqueId="ebba4eee-4b5d-4399-8e9f-77602195d28c">
+			<Documentation>Ground surface of the Site indicating final conditions modelled as _IfcGeomodel_.</Documentation>
+			<Concepts>
+				<DocTemplateUsage Name="Product Local Placement" UniqueId="52718638-0df6-4826-a4ae-cc5ec683a410">
+					<Documentation>Surface Proposed shall be locally placed in the Site.</Documentation>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml:197]:
```suggestion
					<Documentation>Surface Proposed shall be locally placed in the site.</Documentation>
```

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 15]
```diff
@@ -73,5 +99,137 @@
...
+			<Concepts>
+				<DocTemplateUsage Name="Product Local Placement" UniqueId="fcd93046-3f50-4811-8df0-7a0575188e8d">
+					<Documentation>Surface Proposed shall be locally placed in the Site.</Documentation>
+					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Product_Local_Placement_3Bw5jVUH9AGukt7cEGB9j" />
+					<Items>
+						<DocTemplateItem UniqueId="52a2105e-9d63-46e3-a4c6-b88b707a2364" RuleParameters="RelativeToElement=IfcSite;ElementName=HDMI Plattrambro;" />
+					</Items>
+				</DocTemplateUsage>
+				<DocTemplateUsage Name="Geometry" UniqueId="fec732b0-336c-4139-aeaf-9c0e419136b5">
+					<Documentation>Surface Proposed shall be represented as a 3D surface tesselation _IfcTriangulatedIrregularNetwork_ geometric items.</Documentation>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]:
```suggestion
					<Documentation>Surface Proposed shall be represented as a 3D surface tessellation _IfcTriangulatedIrregularNetwork_ geometric items.</Documentation>
```

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 16]
```diff
@@ -73,5 +99,137 @@
...
+					</Items>
+				</DocTemplateUsage>
+			</Concepts>
+			<ApplicableTemplate xsi:type="DocTemplateDefinition" UniqueId="5850bc7e-a2d5-4139-a919-96cf3ce3b66f" Type="IfcGeomodel" />
+		</DocConceptRoot>
+		<DocConceptRoot Name="Surface Existing Geotechnical Stratum" UniqueId="ff6a0881-d581-4823-bf99-5de40cf90fc7">
+			<ApplicableEntity xsi:type="DocEntity" xsi:nil="true" href="IfcGeotechnicalStratum" />
+			<Concepts>
+				<DocTemplateUsage Name="Product Local Placement" UniqueId="fcd93046-3f50-4811-8df0-7a0575188e8d">
+					<Documentation>Surface Proposed shall be locally placed in the Site.</Documentation>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]:
```suggestion
					<Documentation>Surface Proposed shall be locally placed in the site.</Documentation>
```

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 17]
```diff
@@ -73,5 +99,137 @@
...
+					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Surface_Tessellation_Geometry_0gcN00aHT7P84eQ4tB7HA6" />
+					<Items>
+						<DocTemplateItem UniqueId="c4746c74-b98a-436e-80d7-48789063752e" RuleParameters="Items=IfcTriangulatedIrregularNetwork;" />
+					</Items>
+				</DocTemplateUsage>
+			</Concepts>
+			<ApplicableTemplate xsi:type="DocTemplateDefinition" UniqueId="f3811f64-c132-4240-89da-fa649f266b5d" Type="IfcGeomodel" />
+		</DocConceptRoot>
+		<DocConceptRoot Name="Surface Proposed Geotechnical Stratum" UniqueId="49dde5fa-446a-4d74-8b37-dbee334423c7">
+			<Documentation>Ground surface of the Site indicating final conditions modelled as _IfcGeotechnicalStratum_.</Documentation>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml:214]: Mismatch between `DocConceptRoot.Name` and `DocConceptRoot.Documentation` - Surface *Proposed* vs. *final* conditions?

**Review by pjanck — CHANGES_REQUESTED** [review]
Consistency improvements + mismatch between *Proposed* and *Existing* - I suggest a review of all documentation in this PR.

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml (hunk 8 above)
Comment (SergejMuhic) [reply IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]: I think I should do it the other way around. So everything capitalized.

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml (hunk 17 above)
Comment (SergejMuhic) [reply IFC4x3/ModelViews/TI-Bridge/DocModelView.xml:214]: That is how it is defined in the requirements analysis.

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 18]
```diff
@@ -73,5 +99,136 @@
...
+			<Concepts>
+				<DocTemplateUsage Name="Product Local Placement" UniqueId="fcd93046-3f50-4811-8df0-7a0575188e8d">
+					<Documentation>Surface Proposed shall be locally placed in the Site.</Documentation>
+					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Product_Local_Placement_3Bw5jVUH9AGukt7cEGB9j" />
+					<Items>
+						<DocTemplateItem UniqueId="52a2105e-9d63-46e3-a4c6-b88b707a2364" RuleParameters="RelativeToElement=IfcSite;ElementName=HDMI Plattrambro;" />
+					</Items>
+				</DocTemplateUsage>
+				<DocTemplateUsage Name="Geometry" UniqueId="fec732b0-336c-4139-aeaf-9c0e419136b5">
+					<Documentation>Surface Proposed shall be represented as a 3D surface tessellation _IfcTriangulatedIrregularNetwork_ geometric items.</Documentation>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]: Coming from the RAR:

```suggestion
		<DocConceptRoot Name="Surface Existing Geotechnical Stratum" UniqueId="ff6a0881-d581-4823-bf99-5de40cf90fc7">
			<ApplicableEntity xsi:type="DocEntity" xsi:nil="true" href="IfcGeotechnicalStratum" />
			<Concepts>
				<DocTemplateUsage Name="Product Local Placement" UniqueId="fcd93046-3f50-4811-8df0-7a0575188e8d">
					<Documentation>Surface Existing shall be locally placed in the Site.</Documentation>
					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Product_Local_Placement_3Bw5jVUH9AGukt7cEGB9j" />
					<Items>
						<DocTemplateItem UniqueId="52a2105e-9d63-46e3-a4c6-b88b707a2364" RuleParameters="RelativeToElement=IfcSite;ElementName=HDMI Plattrambro;" />
					</Items>
				</DocTemplateUsage>
				<DocTemplateUsage Name="Geometry" UniqueId="fec732b0-336c-4139-aeaf-9c0e419136b5">
					<Documentation>Surface Existing shall be represented as a 3D surface tessellation _IfcTriangulatedIrregularNetwork_ geometric items.</Documentation>
```

File: IFC4x3/ModelViews/TI-Bridge/DocModelView.xml [hunk 19]
```diff
@@ -73,5 +99,136 @@
...
+			<Concepts>
+				<DocTemplateUsage Name="Product Local Placement" UniqueId="437748ef-61eb-485d-92d2-7cdc615a1805">
+					<Documentation>Surface Proposed shall be locally placed in the Site.</Documentation>
+					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Product_Local_Placement_3Bw5jVUH9AGukt7cEGB9j" />
+					<Items>
+						<DocTemplateItem UniqueId="db040752-8fee-45c7-ba17-78eeaa936a13" RuleParameters="RelativeToElement=IfcSite;ElementName=HDMI Plattrambro;" />
+					</Items>
+				</DocTemplateUsage>
+				<DocTemplateUsage Name="Geometry" UniqueId="35622964-9cc2-47ae-aeb1-e465fcb77330">
+					<Documentation>Surface Proposed shall be represented as a 3D surface tessellation _IfcTriangulatedIrregularNetwork_ geometric items.</Documentation>
```
Comment (pjanck) [inline IFC4x3/ModelViews/TI-Bridge/DocModelView.xml]: From RAR:

```suggestion
		<DocConceptRoot Name="Surface Existing Geomodel" UniqueId="7db57e42-f8fb-454c-aa0d-5f803c837b8c">
			<ApplicableEntity xsi:type="DocEntity" xsi:nil="true" href="IfcGeomodel" />
			<Concepts>
				<DocTemplateUsage Name="Product Local Placement" UniqueId="437748ef-61eb-485d-92d2-7cdc615a1805">
					<Documentation>Surface Existing shall be locally placed in the Site.</Documentation>
					<Definition xsi:type="DocTemplateDefinition" xsi:nil="true" href="Product_Local_Placement_3Bw5jVUH9AGukt7cEGB9j" />
					<Items>
						<DocTemplateItem UniqueId="db040752-8fee-45c7-ba17-78eeaa936a13" RuleParameters="RelativeToElement=IfcSite;ElementName=HDMI Plattrambro;" />
					</Items>
				</DocTemplateUsage>
				<DocTemplateUsage Name="Geometry" UniqueId="35622964-9cc2-47ae-aeb1-e465fcb77330">
					<Documentation>Surface Existing shall be represented as a 3D surface tessellation _IfcTriangulatedIrregularNetwork_ geometric items.</Documentation>
```

**Review by pjanck — CHANGES_REQUESTED** [review]
Two new typos, one old one, otherwise good to go!

**Pull request merged**